index.html

```diff
@@ -543,6 +526,15 @@ <h2>Sharing theme</h2>
             this.levelEnd();
           }
         },
+        checkEmojiGuess() {
+          const emoji = emojis[this.currentWord];
+          const alreadyFound = this.unlockedEmojis.some(
+            (entry) => entry.word === this.currentWord
```

Comment:
Should we have something for emojis you've alreadyFound?
I guess the point is to reward new emojis.

Reply:
Yeah, I'm not sure. Kids tried "cats" this morning but didn't seem to mind too much when it didn't come up (again)